src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Services/SyntaxFacts/AbstractSyntaxFacts.cs

```diff
@@ -508,6 +508,22 @@ protected int GetStartOfNodeExcludingAttributes(SyntaxNode root, SyntaxNode node
 
         public abstract SyntaxList<SyntaxNode> GetAttributeLists(SyntaxNode node);
 
+        public abstract bool IsParameterNameXmlElementSyntax(SyntaxNode node);
+
+        public abstract SyntaxNode[] GetContentFromDocumentationCommentTriviaSyntax(SyntaxTrivia trivia);
+
+        public bool IsAwaitKeyword(SyntaxToken token)
+            => token.RawKind == SyntaxKinds.AwaitKeyword;
+
+        public bool IsIdentifier(SyntaxToken token)
+            => token.RawKind == SyntaxKinds.IdentifierToken;
+
+        public bool IsGlobalNamespaceKeyword(SyntaxToken token)
+            => token.RawKind == SyntaxKinds.GlobalKeyword;
+
+        public bool IsHashToken(SyntaxToken token)
+            => token.RawKind == SyntaxKinds.HashToken;
```

Comment:
this should move to ISyntaxKinds

Reply:
I don't know why these 4 things were here. They were unused. Deleted now.